FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +20,40 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let this = self else { return }
+			switch result {
+			case .success((let data, let response)):
+				if response.statusCode == this.ok_200, let feedImages = try? JSONDecoder().decode(FeedImageParser.self, from: data).feedImages {
```

Comment:
Following from my previous comments, by making `ok_200` a static property we could access it without referencing `self`:
```suggestion
				if response.statusCode == RemoteFeedLoader.ok_200, let feedImages = try? JSONDecoder().decode(FeedImageParser.self, from: data).feedImages {
```